platform/commonUI/general/src/controllers/TimeRangeController.js

```diff
@@ -243,36 +211,12 @@ define(
                 updateTicks();
             }
 
-            function updateStartFromText(value) {
-                try {
-                    updateOuterStart(parseTimestamp(value));
-                    updateBoundsTextForProperty($scope.ngModel, 'end');
-                    $scope.boundsModel.startValid = true;
-                } catch (e) {
-                    $scope.boundsModel.startValid = false;
-                    return;
-                }
-            }
+            function updateFormat(key) {
+                formatter = formatService.getFormat(key) ||
```

Comment:
I see two cases here:

* Format has not been specified. I'd like to handle this case by falling back to `utc` as a default; this is what will apply to the majority of telemetry data. Thinking of the developer who is just trying to get some "hello, world" telemetry going, don't want them to get stifled by one missing property. Considering this more, though, I think it would make more sense to supply these defaults from `TelemetryCapability` so that other components can just assume the property will be there.
* Format has been specified but is not recognized. Think you're right that we shouldn't _just_ fall back to UTC here; I see two options:
  * Log a warning and fall back to the default.
  * Throw an error and just fail.

I like falling back to a warning because it's more fault-tolerant, but on consideration I think there is a correctness concern there - you'd be showing data as dates when it's really meant to be displayed as some other thing entirely.

Think this is true for `DateTimeFieldController` as well.